metadoc-core/src/main/protobuf/metadoc.proto

```diff
@@ -16,5 +16,5 @@ message Symbol {
 
 message Index {
     repeated string files = 1;
-    repeated string symbols = 2;
+    repeated Symbol symbols = 2;
```

Comment:
When changing the schema the convention is to bump up the tag id so it becomes

```proto
reserved 2; // this tag id will be ignored when reading old .index files.
repeated Symbol symbols 3;
```

It's not an issue here since there are no binary compatibility to worry about yet.